cdap-ui/app/styles/themes/cdap/header.less

```diff
@@ -1,145 +1,226 @@
 @import "../../variables.less";
 @import "../../../../bower_components/bootstrap/less/mixins.less";
 
-body.theme-cdap header {
-  background-color: @cdap-header;
-
-  [my-navbar] {
-
-    padding-top: 10px;
-    padding-bottom: 15px;
-
-    .navbar-brand {
-      position: relative;
-      top: -5px;
-
-      > strong {
-        display: inline-block;
-        visibility: hidden;
-        width: 84px;
+body {
+  header [my-navbar] {
```

Comment:
If this are the set of styles that will be common across themes then probably we should move this to common.less?